src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -81,6 +81,12 @@
     private WeakReference<SctpConnection> sctpConnection
         = new WeakReference<>(null);
 
+    /**
+     * The {@link WebRtcDataStream} currently used by this
+     * {@link EndpointMessageTransport} instance for writing.
+     */
+    private WebRtcDataStream webrtcDataStream;
```

Comment:
Non-blocking: Maybe add a prefix to indicate that this is the "writable" WebRTC data stream?

Reply:
sure